mtp_cashbook/templates/disbursements/recipient-contact.html

```diff
@@ -36,16 +38,11 @@ <h1 class="heading-xlarge mtp-heading--compact">
             <input id="{{ field.id_for_label }}" class="form-control {% if field.errors %}form-control-error{% endif %} mtp-address-line" name="{{ field.html_name }}" value="{{ field.value|default:'' }}" type="{{ field.field.widget.input_type|default:'text' }}" />
           </div>
         {% endwith %}
-
-        {% with field=form.city %}
-          {% include 'mtp_common/forms/field-errors.html' with field=field only %}
-          <div>
-            <input id="{{ field.id_for_label }}" class="form-control {% if field.errors %}form-control-error{% endif %} mtp-address-city" name="{{ field.html_name }}" value="{{ field.value|default:'' }}" type="{{ field.field.widget.input_type|default:'text' }}" />
-          </div>
-        {% endwith %}
       </div>
     </fieldset>
 
+    {% include 'mtp_common/forms/field.html' with field=form.city input_classes='mtp-postcode' value=form.city.value only %}
```

Comment:
this field used to have `mtp-address-city` as the class, i don't think it needs switching